src/Microsoft.IdentityModel.Tokens/CryptoProviderFactory.cs

```diff
@@ -41,6 +41,8 @@ public class CryptoProviderFactory
         private static CryptoProviderFactory _default;
         private static ConcurrentDictionary<string, string> _typeToAlgorithmMap = new ConcurrentDictionary<string, string>();
         private static object _cacheLock = new object();
+        private static int _defaultSignatureProviderObjectPoolCacheSize = Environment.ProcessorCount * 4;
```

Comment:
Just curios on why we decided to change the default object pool size?
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/26396d50b996d9e19cda286f614b58068aa4528f/src/Microsoft.IdentityModel.Tokens/ObjectPools.cs#L63